city_scrapers/spiders/chi_ohare_noise.py

```diff
@@ -0,0 +1,183 @@
+from city_scrapers_core.constants import NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from scrapy.selector import Selector
+import requests
+import datetime
+
+
+class ChiOhareNoiseSpider(CityScrapersSpider):
+    name = "chi_ohare_noise"
+    agency = "Chicago O'Hare Noise Compatibility Commission"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.oharenoise.org"]
+    meetings = dict()
+    start_urls = ["https://www.oharenoise.org/about-oncc/agendas-and-minutes"]
+
+    def parse(self, response):
+
+        """
+        `parse` should always `yield` Meeting items.
+        """
+        table_rows = response.selector.xpath(".//tbody//tr")
+
+        for item in table_rows:
+            date = self._parse_start(item)
+            url_date = date.strftime("%Y/%m/%d")
+            meeting = Meeting(
+                title=self._parse_title(item),
+                links=self._parse_links(item),
+                start=self._parse_start(item),
+                source=self._parse_source(response)
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+            self.meetings[url_date] = meeting
+
+        self._parse_months()
+        for date, meeting in self.meetings.items():
+            yield meeting
```

Comment:
There are still several missing fields on the `Meeting` objects when they're yielded here. `all_day` and `classification` are two of them, and while `all_day` can be `False`, `_parse_classification` will need to be implemented

Reply:
Gotchya will do!

Reply:
So I'm a little confused about how to classify each meeting, is there a specific formula to determine which classification meetings should be based on their title?

Reply:
For now I'm just going to parse the title for the classification constants and define them that way. Let me know if this is ok. Having fun working on this!

Reply:
We've got a list of the classification constants and what they mean here https://cityscrapers.org/docs/development/#classifications, but usually it lines up with the titles